crates/store/re_log_encoding/src/lib.rs

```diff
@@ -121,6 +122,9 @@ pub enum OptionsError {
     #[error("Unknown compression: {0}")]
     UnknownCompression(u8),
 
+    #[error("Attempted to use the removed MsgPack serializer, which is no longer supported")]
```

Comment:
```suggestion
    #[deprecated(since = "1.0.0")]
    #[error("Attempted to use the removed MsgPack serializer, which is no longer supported")]
```

Maybe we should start adding breadcrumbs for ourselves to remind us to remove these things?

Reply:
Agreed, but not sure about `deprecated`. That just turns into `#[expect(deprecated)]` wherever it's used, which in this case is the same file. I'll add a comment